accepted/PSR-16-simple-cache.md

```diff
@@ -215,7 +216,8 @@ interface CacheInterface
      *
      * @throws \Psr\SimpleCache\InvalidArgumentException
      *   MUST be thrown if $values is neither an array nor a Traversable,
-     *   or if any of the $values are not a legal value.
+     *   if any of the $values are not a legal value
+     *   OR if $ttl is neither an integer or a \DateInterval object.
```

Comment:
This one is a valid clarification of the docblocks I guess but I don't think the spec can be changed that it's been voted in..